.github/workflows/binaries.yml

```diff
@@ -92,3 +101,53 @@ jobs:
             name: '${{ env.NAME }}.tar.gz',
             data: fs.readFileSync('target/${{ matrix.sys.target }}/release/${{ env.NAME }}.tar.gz'),
           });
+
+  installer:
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Dependencies
+        run: sudo apt-get update && sudo apt-get -y install nsis wget
+
+      - name: Setup vars
+        run: |
+          version="$(cargo metadata --format-version 1 --no-deps | jq -r '.packages[] | select(.name == "stellar-cli") | .version')"
+          echo "VERSION=${version}" >> $GITHUB_ENV
+          echo "STELLAR_CLI_INSTALLER=stellar-cli-installer-${version}-x86_64-pc-windows-msvc.exe"
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: stellar-cli-${{ env.VERSION }}-x86_64-pc-windows-msvc.tar.gz
+          path: stellar-cli.tar.gz
+
+      - name: Uncompress Artifact
+        run: tar xvf stellar-cli.tar.gz
+
+      - name: Build Installer
+        run: |
+          makensis -V4 -DSTELLAR_CLI_INSTALLER="${{ env.STELLAR_CLI_INSTALLER }}" -DSTELLAR_CLI_VERSION="${{ env.VERSION }}" installer.nsi
```

Comment:
Can we add in a use of `actions/upload-artifact@v4` to upload the artifact to the GitHub build? It allows us to download it and use it in PRs to either debug it or use it, even if a release hasn't occurred.

Reply:
Done!